src/main.js

```diff
@@ -24,7 +24,7 @@ import {
 import {
   faLinkedin,
   faFacebookSquare,
-  faInstagram
+  faInstagramSquare
```

Comment:
Are all the places previously using `faInstagram` updated?

Reply:
Tip:
![image](https://user-images.githubusercontent.com/44103628/81081968-757c8980-8ea7-11ea-9225-c17af2af156b.png)